packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -490,6 +497,12 @@ contract Deploy is Deployer {
         save("ProtocolVersions", address(versions));
         console.log("ProtocolVersions deployed at %s", address(versions));
 
+        Types.ContractSet memory contracts = _proxiesUnstrict();
+        contracts.ProtocolVersions = address(versions);
```

Comment:
Just leaving a comment to record the decision we reached, which was to add a comment to explain wherever we do these overrides.